test/core/test_serde.py

```diff
@@ -188,3 +189,13 @@ def test_timestamp_encode_decode() -> None:
 )
 def test_string_escape(serialize_fn) -> None:
     assert serialize_fn(r"a\b") == r"a\b"
+
+
+def test_serde_fq():
+    serde.encode(test_serde_fq)
```

Comment:
Is this testing something? I guess we should have a test running `decode(encode(f))` for some function `f` to check that it reconstructs it, right?